admin-ui/app/template/css-component/Css.tsx

```diff
@@ -7,11 +7,6 @@ const Css = ({ menuList }: any) => {
   const cssP = components[1].map((general: any) =>
```

Comment:
what is [1], you must have proper item loop, if it is first item ... then user array fist item function